Lib/copy.py

```diff
@@ -74,20 +76,20 @@ def copy(x):
         # treat it as a regular class:
         return _copy_immutable(x)
 
-    copier = getattr(cls, "__copy__", None)
-    if copier is not None:
+    copier = getattr(cls, "__copy__", NoValue)
```

Comment:
I'm not sure this is worth changing just for consistency with pickle. There's some real-world code relying on the current behavior: https://github.com/hikari-py/hikari/blob/master/hikari/impl/special_endpoints.py#L1721

Reply:
@JelleZijlstra I am not sure either. I am fine with removing this from this PR (not just this line, but then also the other changes in the `copy.py`).  If we decide later we want to make these changes, we can open a new PR.